src/main/scala/is/hail/methods/LDPrune.scala

```diff
@@ -315,18 +370,17 @@ object LDPrune {
       if (gpi.persist) gpi.rdd.persist(StorageLevel.MEMORY_AND_DISK)
```

Comment:
Should this be a `persist2`?

Reply:
These are GeneralRDD, so they don't have persist2, unpersist2. I think I need to implement persist2, unpersist2 on a GeneralRDD[RegionValue]. I'm looking into this now. Let me know if you have thoughts/suggestions.